docs/tutorials/faq/main.md

```diff
@@ -12,82 +12,83 @@ right answer.
-      + [Can it route SIP instant messaging requests (page mode, RFC3428)?](#can-it-route-sip-instant-messaging-requests-page-mode-rfc3428)
-      + [Can it store instant messages for offline users and deliver them when the users become online?](#can-it-store-instant-messages-for-offline-users-and-deliver-them-when-the-users-become-online)
-      + [Can it route instant messages using MSRP (RFC4975)?](#can-it-route-instant-messages-using-msrp-rfc4975)
-      + [Can it handle end-to-end presence requests?](#can-it-handle-end-to-end-presence-requests)
-      + [Does it have support for SIP SIMPLE Presence (presence agent model)?](#does-it-have-support-for-sip-simple-presence-presence-agent-model)
-      + [Does it have support for Resource Lists Service?](#does-it-have-support-for-resource-lists-service)
-      + [Can it act as a presence user agent?](#can-it-act-as-a-presence-user-agent)
-      + [Can it handle subscription requests for dialog states (aka, blinking lamps)?](#can-it-handle-subscription-requests-for-dialog-states-aka-blinking-lamps)
-      + [Does it have support for handling XCAP?](#does-it-have-support-for-handling-xcap)
-   * [NAT Traversal](#nat-traversal)
-      + [Can Kamailio be used with phones connecting from behind NAT?](#can-kamailio-be-used-with-phones-connecting-from-behind-nat)
-      + [Can Kamailio be installed behind NAT (e.g., like on Amazon EC2)?](#can-kamailio-be-installed-behind-nat-eg-like-on-amazon-ec2)
-   * [Configuration File Processing](#configuration-file-processing)
-      + [Can I check the configuration files for syntax errors?](#can-i-check-the-configuration-files-for-syntax-errors)
-      + [Do I have to delete the AVPs added in the configuration file for a SIP message?](#do-i-have-to-delete-the-avps-added-in-the-configuration-file-for-a-sip-message)
-      + [What is a pseudo-variable?](#what-is-a-pseudo-variable)
-      + [How to iterate through the items in a comma separated string?](#how-to-iterate-through-the-items-in-a-comma-separated-string)
-      + [How is the function return code evaluated?](#how-is-the-function-return-code-evaluated)
-      + [How is the SIP request retransmission handled?](#how-is-the-sip-request-retransmission-handled)
-   * [Troubleshooting](#troubleshooting)
-      + [I get 'out of memory' errors, what can be done?](#i-get-out-of-memory-errors-what-can-be-done)
-   * [Tools](#tools)
-      + [How to generate HA1 value in command line?](#how-to-generate-ha1-value-in-command-line)
-      + [How to generate HA1B value in command line?](#how-to-generate-ha1b-value-in-command-line)
-      + [Kamctl gives an error regarding opening FIFO file, why?](#kamctl-gives-an-error-regarding-opening-fifo-file-why)
-      + [Kamcmd sometime fails to execute RPC commands with number parameters, why?](#kamcmd-sometime-fails-to-execute-rpc-commands-with-number-parameters-why)
+  - [Table Of Contents](#table-of-contents)
```

Comment:
I don't think an entry inside ToC has to point to ToC/itself, I haven't seen it in any book so far.